app/models/user.rb

```diff
@@ -2,8 +2,13 @@ class User < ApplicationRecord
   validates :first_name, presence: true, length: { maximum: 50 }
   validates :last_name, presence: true, length: { maximum: 50 }
   validates :email, format: { with: URI::MailTo::EMAIL_REGEXP }, uniqueness: true, length: { maximum: 255 }
-
+  validates :mobile, presence: true,
+                     numericality: true,
+                     length: { minimum: 10, maximum: 20 }
   before_save :downcase_email
+  enum role: { admin: 0, lead: 1, members: 2, simple: 3 }
```

Comment:
make a constant list for roles

check how company types were made

Reply:
ok